### ISSUE TITLE
[Consistency] User encounters design components that are inconsistent or confusing.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80608]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User encounters design components that are inconsistent or confusing.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On a /document-request page (eg. [test user Kyle Cole/+23, /document-request/467558](https://staging.va.gov/track-claims/your-claims/600471552/document-request/467558), the "Need to mail your files?" additional info is presented before the file upload button. But on the general /files Claim Files page, "Need to mail your files?" comes after both the file upload and the Submit files for review button.

## Recommended action

These should be placed in a relatively consistent place, and I think before the file upload makes the most sense. Screen reader users are likely to think of the submit button as the end of their interaction --- meaning they're not likely to go looking for more information about how to submit files after the submit button.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jacobworrell** [comment]
@skylerschain please check whether this link is still situated in different places between evidence request and files tabs.

**Comment by skylerschain** [comment]
It's below the "Submit files for review" button in the Files tab, as well as HAIMS request (see below)

<img width="557" alt="Screenshot 2024-07-16 at 10 42 01 AM" src="https://github.com/user-attachments/assets/4b3ccf13-3c5b-45b7-8791-5e8ed0ea9105">
<img width="726" alt="Screenshot 2024-07-16 at 10 44 21 AM" src="https://github.com/user-attachments/assets/85949750-e659-4d77-b9fc-8cae318213c0">

I don't see it for 5103 Document Request page. I think the mock didn't include it so we would have to create a ticket to add that.

**Comment by davidakennedy** [comment]
@skylerschain I think it was user 228, but I can't log in with that user right now for some reason.

**Comment by jacobworrell** [comment]
@shiragoodman The "Need to mail your files" content is now consistently underneath the document submit button.

**Comment by shiragoodman** [comment]
@briandeconinck FYSA
